package.json

```diff
@@ -42,14 +42,11 @@
     "@rollup/plugin-node-resolve": "^13.2.1",
     "@rollup/plugin-replace": "^4.0.0",
     "@sheerun/mutationobserver-shim": "^0.3.3",
-    "@storybook/addon-a11y": "^7.6.17",
-    "@storybook/addon-actions": "^7.6.17",
-    "@storybook/addon-controls": "^7.6.17",
```

Comment:
The actions, controls, and viewport addons are bundled into the "essentials" addon and so we don't need to depend on them separately. 

https://storybook.js.org/integrations/tag/essentials